src/seedu/addressbook/data/tag/Tag.java

```diff
@@ -12,6 +14,7 @@
     public static final String TAG_VALIDATION_REGEX = "\\p{Alnum}+";
 
     public final String tagName;
+    public Set<Tagging> taggings;
 
```

Comment:
Take a look at the class diagram again. Addressbook should keep track of the list of taggings.